web/src/types.ts

```diff
@@ -146,6 +146,7 @@ export interface MapGeometry extends Feature<Polygon | MultiPolygon> {
   properties: {
     zoneId: string;
     color: string;
+    center?: [number, number];
```

Comment:
I don't think this need to be optional, there should always be a center for each zone.